.github/workflows/build.yaml

```diff
@@ -15,6 +15,11 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
+      - name: Free disk space
+        uses: jlumbroso/free-disk-space@main
+        with:
+          large-packages: false
```

Comment:
In libra-framework removing large packages causes some build errors, but I didn't investigate much further than that.